README.md

```diff
@@ -1,20 +1,12 @@
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
+# WhatNow (V0.0)
 
-# Address Book (Level 4)
+<img src="docs/images/UI_Prototype/WhatNowUI-Welcome-Colour.png" width="600"><br>
 
-<img src="docs/images/Ui.png" width="600"><br>
+Have you ever missed a deadline because you forgot to write it down? Have you ever forgot that you have to do something?
```

Comment:
how does your application help if we forgot to record the activity in the first place?